.github/workflows/pytorch-version-tests.yml

```diff
@@ -5,6 +5,7 @@ on:
   schedule:
     # Run at 00:00 UTC Every Day
     - cron: "0 0 * * *"
+  workflow_dispatch: 
```

Comment:
@trsvchn do you think we would need that in future ?

Reply:
I think no, its for manual triggering, and we definitely don't want that, because of number of tests

Reply:
This is a simple trigger to allow you to run workflows manually.

Reply:
@gucifer thats interesting, i've never used it before, so who has permissions for manual triggering?

Reply:
@gucifer let's remove this from here. If we think to add it, then let's do this in another PR.

Reply:
ANyone with collabrator rights or above can do it